services/api/src/main/java/org/eclipse/osc/services/api/OrchestratorApi.java

```diff
@@ -33,21 +36,16 @@ public Response fetch(@HeaderParam("ocl") String oclLocation) throws Exception {
     @Path("/health")
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String health() throws Exception {
-        if (getOrchestrator() == null) {
-            throw new IllegalStateException("Orchestrator service is not ready");
-        }
+    public String health() {
```

Comment:
Why removing the check on the orchestrator service presence here ?

Reply:
This check is already there in the getOrchestrator() method. That's why removed it.

Reply:
Ah yes, good point. I agree then.